src/properties.go

```diff
@@ -29,6 +29,8 @@ const (
 	AlwaysEnabled Property = "always_enabled"
 	// SegmentTemplate is the template to use to render the information
 	SegmentTemplate Property = "template"
+	// VersionUrlTemplte i the template to use when building language segment hyperlink
```

Comment:
typo ;-)

Reply:
that damned butterfly keyboard :sweat_smile: